docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -47,15 +47,41 @@ handling user interactions, and managing search functionality.
 Actions are an optional way to allow users to perform custom tasks such as
 adding a new item to the list of options.
 
-#### Callback
+### Selection Behavior
 
-`onSelect` is the recommended callback handler that should be used in most cases. This will immediately inform
-the consumer of changes to selections as they happen.
+#### Single Select
 
-If however immediate changes are not desired, and instead it is preferable to wait until all selections or
-modifications to selections have been made before informing the consumer, `onClose` may be used. The arguments provided to this callback are
-Identical to those of `onSelect`, only the timing differs. Please note this will only
-have a noticeable effect with `multiSelect = true` since the Combobox closes immediately upon selection with single-select.
+This is the default. When a selection is made, the Combobox will close.
+
+#### Multi Select
+
+Add the `multiSelect` prop for this behavior. This enables multiple options to
+be selected and/or un-selected, and will only close when a user takes an
+explicit action to close the open Combobox. eg. clicking outside the Combobox,
+or pressing the ESC key.
```

Comment:
@chris-at-jobber  when we were moving over the documentation, I had asked the team how they felt about not putting the component name in backticks.  Only reserving the backticks for props.  
It's understandable that since all the docs pages were written by various people, they all had different patterns but in an effort to streamline doc page headings, I wanted to streamline the use of backticks as well.

I bring this up here because we mention Combobox within this paragraph and it's easy to miss backticking that word.  The pattern is already broken. 

I would personally rather read through documentation that looks like this:

![Screenshot 2023-10-16 at 2 47 07 PM](https://github.com/GetJobber/atlantis/assets/104797704/51feafa5-8879-41c3-9a04-a7334a041319)

Compared to this example which personally, halts the flow of me reading through the sentence:

![Screenshot 2023-10-16 at 2 44 15 PM](https://github.com/GetJobber/atlantis/assets/104797704/9f74616b-7404-4c3f-9523-4fdd37c24fa0)

I feel pretty strongly that we don't need to highlight the component name because we know which component docs we are in.  Highlighting the props is helpful because your eye is drawn to those words if you need them.

That being said, I recognize this is a personal preference so will follow which ever way we think is best. I'm really curious to hear what you think, Chris.

Reply:
I think that makes sense! I don't think anyone had ever talked about it explicitly before and I know even in a given component we'll have a mix of, say, `Button` and Button 😄

Reply:
Perfect! Thanks Chris!